pull

```diff
@@ -36,7 +36,7 @@
 git submodule update
 
 # Make the `config` dir current.
-cd ./config
+cd ./config || { echo "Error: `config` directory is missing."; exit 1; }
```

Comment:
The backticks should not be used in shell scripts. They are used as a deprecated way of evaluating variables. (see https://github.com/koalaman/shellcheck/wiki/SC2006)

Reply:
Fixed. Also removed `.bat` file.